packages/terra-search-field/src/SearchField.jsx

```diff
@@ -250,8 +263,13 @@ class SearchField extends React.Component {
       customProps.className,
     );
 
-    const inputText = inputAttributes && Object.prototype.hasOwnProperty.call(inputAttributes, 'aria-label') ? inputAttributes['aria-label'] : intl.formatMessage({ id: 'Terra.searchField.search' });
+    const labelClassNames = cx([
+      'label',
+      theme.className,
+    ]);
 
+    const inputAriaLabelText = inputAttributes && Object.prototype.hasOwnProperty.call(inputAttributes, 'aria-label') ? inputAttributes['aria-label'] : intl.formatMessage({ id: 'Terra.searchField.search' });
+    const groupNameValue = groupName === 'Search' ? intl.formatMessage({ id: 'Terra.searchField.search' }) : groupName;
```

Comment:
Can't search be internationalized?  The groupName is a prop to be displayed to the user.  I would expect that we would only use our message if groupName is undefined.  However, that could just be the default prop in that case.

Reply:
What I am doing here is internationalizing that default prop value which is `Search`. I do not have access to intl object when defining default props, hence this code here. 
Basically, If there is a custom user defined groupName, that would take precedence. Otherwise, it would be the default internationalized value. 
Does that make sense?